logger/scenario_logger/include/scenario_logger/logger.h

```diff
@@ -114,8 +114,8 @@ const rclcpp::Time& begin();
 std::string toIso6801(const rclcpp::Time& stamp);
 
 boost::property_tree::ptree toJson(const scenario_logger_msgs::msg::MetaData& data);
-boost::property_tree::ptree toJson(const scenario_logger_msgs::msg::LoggedData& data);
-boost::optional<boost::property_tree::ptree> toJson(const scenario_logger_msgs::msg::Log& data,
+boost::optional<boost::property_tree::ptree> toJson(const scenario_logger_msgs::msg::Log& data);
```

Comment:
Easy to get lost here because the naming is so confusing, we have `Log& data`, `LoggedData& data`, `Logger`, `Logger log` etc.